api/src/main/java/eu/darkbot/api/managers/MemoryAPI.java

```diff
@@ -11,8 +11,16 @@
  * @deprecated Plugins should not rely on modifying memory to access functions, and should use other APIs instead.
  */
 @Deprecated
```

Comment:
let's avoid extending deprecated classes. Kill this off if necessary and put this in the bot directly instead